packages/edit-widgets/src/components/save-button/index.js

```diff
@@ -1,19 +1,45 @@
+/**
+ * External dependencies
+ */
+import { filter, map, some, forEach } from 'lodash';
+
 /**
  * WordPress dependencies
  */
 import { Button } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
-import { useDispatch } from '@wordpress/data';
-import { useState, useCallback } from '@wordpress/element';
+import { useCallback } from '@wordpress/element';
+import { useDispatch, useSelect } from '@wordpress/data';
 
 function SaveButton() {
-	const [ isSaving, setIsSaving ] = useState( false );
-	const { saveWidgetAreas } = useDispatch( 'core/edit-widgets' );
-	const onClick = useCallback( async () => {
-		setIsSaving( true );
-		await saveWidgetAreas();
-		setIsSaving( false );
-	}, [] );
+	const { widgetAreaEdits, isSaving } = useSelect(
```

Comment:
`widgetAreaEdits` does not contain edits. Maybe a better name is `editedWidgetAreaIds`.